src/local_scheduler/local_scheduler.cc

```diff
@@ -344,22 +346,18 @@ LocalSchedulerState *LocalSchedulerState_init(
   if (redis_primary_addr != NULL) {
     int num_args;
     const char **db_connect_args = NULL;
```

Comment:
While we're at it, can you make `db_connect_args` a vector so that it can be constructed more cleanly?

Reply:
Yea, I'll have a fix ready soon.